samples/unit-tests/indicator-price-indicator/recalculations/demo.js

```diff
@@ -32,3 +50,47 @@ QUnit.test('Price indicator.', function (assert) {
         'No errors when points are missing.'
     );
 });
+
+QUnit.test('Datagrouping and setExtremes.', function (assert) {
+
+    var chart = Highcharts.stockChart('container', {
+            rangeSelector: {
+                selected: 5
+            },
+            series: [{
+                type: 'candlestick',
+                dataGrouping: {
+                    enabled: true
+                },
+                lastVisiblePrice: {
+                    enabled: true,
+                    label: {
+                        enabled: true
+                    }
+                },
+                lastPrice: {
+                    enabled: true,
+                    color: 'red'
+                },
+                data: getData()
+            }]
+        }),
+        series = chart.series[0];
+
+    assert.ok(
```

Comment:
Should be `assert.strictEqual`. Please read QUnit docs.

Reply:
Sorry, I was confused about replacament form above comment.